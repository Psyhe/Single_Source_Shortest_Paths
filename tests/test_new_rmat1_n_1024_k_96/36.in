1024 396 406
396 784 175
396 156 115
396 128 154
396 517 141
396 532 53
396 1 219
397 176 180
397 12 132
398 288 116
400 347 111
400 8 222
400 96 205
400 97 4
400 100 152
400 40 35
400 358 60
400 4 178
400 5 232
400 104 158
400 545 79
400 32 127
400 532 184
400 800 241
400 2 131
400 139 233
400 512 75
400 1 105
400 101 72
400 129 105
400 260 254
400 0 84
400 268 55
400 32 154
400 70 125
400 384 177
401 320 173
401 12 233
401 520 205
401 241 187
401 612 158
401 337 208
401 2 75
401 16 33
402 0 235
402 68 129
402 528 118
402 64 47
402 12 103
402 676 96
402 73 34
402 521 26
402 212 192
402 6 207
402 64 71
403 172 147
404 259 120
404 1 236
404 0 50
404 112 137
404 4 106
404 32 185
404 288 170
404 18 231
404 0 186
404 512 163
404 312 79
405 129 200
405 512 99
405 524 210
405 0 227
405 333 238
405 0 69
405 592 66
406 576 124
406 328 182
406 8 139
406 53 123
406 577 182
406 33 183
