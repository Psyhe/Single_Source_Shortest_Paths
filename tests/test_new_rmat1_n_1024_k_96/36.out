72
140
123
1e+18
10
38
40
170
50
69
129
